src/components/MultiHopTrade/components/MultiHopTradeConfirm/MultiHopTradeConfirm.tsx

```diff
@@ -142,9 +143,16 @@ export const MultiHopTradeConfirm = memo(() => {
             titleTranslation={
               isArbitrumBridgeWithdraw ? 'bridge.arbitrum.success.tradeSuccess' : undefined
             }
-            descriptionTranslation={
-              isArbitrumBridgeWithdraw ? 'bridge.arbitrum.success.withdrawComplete' : undefined
-            }
+            sellAsset={activeQuote?.steps[0].sellAsset}
+            buyAsset={activeQuote?.steps[0].buyAsset}
+            sellAmountCryptoPrecision={fromBaseUnit(
+              activeQuote?.steps[0].sellAmountIncludingProtocolFeesCryptoBaseUnit,
+              activeQuote?.steps[0].sellAsset.precision ?? 0,
+            )}
+            buyAmountCryptoPrecision={fromBaseUnit(
+              activeQuote?.steps[activeQuote.steps.length - 1].buyAmountAfterFeesCryptoBaseUnit,
+              activeQuote?.steps[activeQuote.steps.length - 1].buyAsset.precision ?? 0,
+            )}
```

Comment:
nitpick: can use `selectLastHop` selector to get the last hop  and keep this cleaner

Reply:
We can also avoid nullish coalescing then for extra safety (check for `lastHop` on l.140 as well), the only reason we default to 0 here is because `activeQuote` can be undefined